### PR TITLE
chore: Bump Sass-loader to 16.0.3

### PR DESCRIPTION
**Pull request by salisdemos**
Since I had to adjust some options, I had to create the PR manually.
For details, why I had to change things see https://webpack.js.org/loaders/sass-loader/#api

### CODE REVIEW

File: client/fe/webpack/moduleRules.js [hunk 1]
```diff
@@ -160,13 +160,16 @@ const moduleRules =
         {
           loader: 'sass-loader',
           options: {
-            implementation: require('sass-embedded'),
-            additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
+            api: 'modern',
```
Comment (hwiem) [inline client/fe/webpack/moduleRules.js:163]: Just wondering: Why not 'modern-compiler', as it is strongly recommended?

> Using modern-compiler and sass-embedded together significantly improve performance and decrease built time. We strongly recommend their use. We will enable them by default in a future major release.

Comment (salisdemos) [reply client/fe/webpack/moduleRules.js:163]: oh no. Something went wrong. These changes where not supposed to be here in this PR.

Comment (salisdemos) [reply client/fe/webpack/moduleRules.js:163]: When I use "modern-compiler" the build doesn't finish. No errors, no output, but it doesn't stop.

File: client/fe/webpack/moduleRules.js [hunk 2]
```diff
@@ -160,13 +160,16 @@ const moduleRules =
...
             sassOptions: {
-              includePaths: [
-                config.projectRoot + 'web/',
-                config.publicPath
-              ]
+              implementation: require('sass-embedded'),
+              additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
+              sassOptions: {
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply client/fe/webpack/moduleRules.js:167]: Thanks for digging this out. I'll have a deeper look

File: client/fe/webpack/moduleRules.js (hunk 1 above)
Comment (salisdemos) [reply client/fe/webpack/moduleRules.js:163]: added the workaround @spiess-demos mentioned here: 2179f0d7194c62e1c74e18fe509a829a52ce53f7

File: client/fe/webpack/moduleRules.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (salisdemos) [reply client/fe/webpack/moduleRules.js]: lets try it: 2179f0d7194c62e1c74e18fe509a829a52ce53f7

File: client/fe/webpack/moduleRules.js [hunk 3]
```diff
@@ -160,10 +160,11 @@ const moduleRules =
         {
           loader: 'sass-loader',
           options: {
-            implementation: require('sass-embedded'),
-            additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
+            api: 'modern-compiler',
             sassOptions: {
-              includePaths: [
+              implementation: require('sass-embedded'),
+              additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
```
Comment (spiess-demos) [inline client/fe/webpack/moduleRules.js:166]: When i tried it like this, the build failed with some error about scss variables inside css files. My suggestion would be to drop this line entirely and hardcode the path in the project that uses the var.

Comment (salisdemos) [reply client/fe/webpack/moduleRules.js:166]: seriousely? My build builds.

Comment (spiess-demos) [reply client/fe/webpack/moduleRules.js:166]: then lets go with it!

**Review by spiess-demos — APPROVED** [review]
lgtm (apart from the non-reproducable-we'll-see-what-happens error).

**Pull request merged**